### DISCUSSION TITLE
Garage Type

### DISCUSSION THREAD

**Discussion by thebonbon** [EveronLife/EveronLife#176] [Polls]
What type of garage do you like more?

1. **Simple garage with a static camera.** 
->Left / right controls to swap, select to take vehicle out

https://user-images.githubusercontent.com/45827986/198717137-80ef4fba-1192-4ad5-b25f-d226e03c7cd8.mp4

2. **Real garage that you can physically enter.**
-> Every player has their own garage instance
-> Get in driver seat to take vehicle out.
Ideas:
-> Different garage types with different amounts of vehicles you can store
-> Little UI to enter GarageID to join other garages

https://user-images.githubusercontent.com/45827986/198717160-32dc13dc-c623-471d-8168-5b6485976564.mp4

**Comment by LuxPoseidon** [comment]
How does the instancing work for number 2? is it under the map? near 0,0,0?

I think both should be added to framework if you are up for it, have a config file where you can swap it to the server owners choice.

**Reply by LuxPoseidon** [reply]
I do like the idea of physical spaces though, perhaps have two different buildings one small one large, upgradability for players to spend money on for a larger space.

**Reply by LuxPoseidon** [reply]
"Little UI to enter GarageID to join other garages" would this be the same as the players id?

**Reply by thebonbon** [reply]
> How does the instancing work for number 2? is it under the map? near 0,0,0?
>
Somewhere where it can't be seen. 0 0 0  or even another world.

>"Little UI to enter GarageID to join other garages" would this be the same as the players id?
>
Something that can lead back to the player id, but input should just be a number like 1234

**Comment by Vergyy** [comment]
Depends on what type of garage it is. Is this like the garage for buying vehicles or is this the type of garage someone stores their vehicles? For storing personal vehicles it should 100% be an enterable garage. This is a life server which immersion plays a huge part. If it is for buying then a static one is fine. That being said a walk in dealership would be very nice.

**Reply by thebonbon** [reply]
> Depends on what type of garage it is. Is this like the garage for buying vehicles or is this the type of garage someone stores their vehicles? For storing personal vehicles it should 100% be an enterable garage. This is a life server which immersion plays a huge part. If it is for buying then a static one is fine. That being said a walk in dealership would be very nice.
>
The poll is about the garage where you can store vehicles but the static camera one could also be used for the vehicle shop where you can select the one you want to buy with a slider at the side for color and maybe some more info about the vehicle, like top speed, inventory size etc..

https://user-images.githubusercontent.com/45827986/199299958-f47420cb-7dc1-442e-b5fa-d2da461a686d.mp4

**Comment by Pyth3rEx** [comment]
Here is an idea, we always saw the ability to sell our cars, but how about sold cars would be purchasable by other players as second hands?
This would allow for poor players access to better vehicles and would put some reasoning behind selling your car.

If #203 where to be added, it could be a good add to itself
